### ISSUE TITLE
Adviod code is unreachable when placing another order - Python

### ISSUE THREAD

**Issue by Oriz1206** [ccxt/ccxt#15153]
- OS: window 10 pro 64bit
- Programming Language version: python 3.
- CCXT version:1.93.3

image1:
![image](https://user-images.githubusercontent.com/74316770/193331000-c3bd2f2a-1cc8-4063-bd0f-59d4d9ef62f3.png)

image2:
![image](https://user-images.githubusercontent.com/74316770/193232983-d45b8773-41b5-4c2f-b392-9601e1710005.png)
Hi!
As the title of the topic does anyone have solution to this problem?

if I using if else stament like it pretty obvious to trigger like this:
image3:
![image](https://user-images.githubusercontent.com/74316770/193330924-79e62e8c-6825-4f51-9717-ce2bf1437d6c.png)
it reachable but it gonna bulky.

how can i reach 2nd order in image1 without making it like image3?

**Comment by carlosmiei** [comment]
Hello @Oriz1206, python is very identation-sensitive, are you sure you don't have any extra space in those statements?

**Comment by Oriz1206** [comment]
> 
Hi, Im sure that there is no extra space or any statements between and they are at the same identation!

**Comment by Oriz1206** [comment]
> Hello @Oriz1206, python is very identation-sensitive, are you sure you don't have any extra space in those statements?

Hi, Im sure that there is no extra space or any statements between and they are at the same identation!

**Comment by carlosmiei** [comment]
@Oriz1206 do you have any error running this snippet in the terminal*

**Comment by Oriz1206** [comment]
> @Oriz1206 do you have any error running this snippet in the terminal*

No, i don't. It work fine when i put it in if else statements like image3 but since I remove statements , it start look like image1. Or put those create order line in to try- except like this:
![image](https://user-images.githubusercontent.com/74316770/193311962-23cf0b5c-48cc-422e-8b75-fdf5bf8a515e.png)
It reachable tho. Seem like those place order line couldn't be placed next to each other, but a block of code will do the trick

**Comment by Stillwind27** [comment]
Having same issue as of yesterday.  Currently using if statement before order creation so that it will work.

**Comment by casperbh96** [comment]
It must be a type-hint bug somewhere. If you correctly type hint your code, you will not get that error.  The problem is that the `ccxt.Exchange` type works as an interface that has a bunch of functions that have not been implemented.

BAD:

```python
import ccxt
exchange: ccxt.Exchange = ccxt.binance()
```

WORK-AROUND (should fix any type hint / code unreachable):

Simply create an array of types in a Union for the exchanges you would use.

```python
import ccxt
from typing import Union
client_types = Union[ccxt.binance, ccxt.binanceusdm]

client = 'Binance'
trading_type = 'futures'

if client == 'Binance' and trading_type == 'futures':
    exchange: client_types = ccxt.binanceusdm()
elif ...

```

**Comment by ttodua** [comment]
this topic has been inactive for several years. I might assume that users no longer face this problem, so closing this issue.
if anyone still experiences the problem, let us know.
